packages/evm/src/binaryTreeAccessWitness.ts

```diff
@@ -415,9 +414,12 @@ export const generateBinaryExecutionWitness = async (
     }
   }
 
-  // Get values from the tree for each stem and suffix
+  // Get values and proofs from the tree for each stem and suffix
   for (const stem of accessedSuffixes.keys()) {
     tree.root(parentStateRoot)
+    // Generate proofs for each stem from prestate root
+    const proof = await tree.createBinaryProof(hexToBytes(stem))
```

Comment:
should we do multiproof instead of per stem proof?

Reply:
Ideally, yes.  We just don't have multiproofs implemented for binary trees yet.  This is as rudimentary as it gets.